Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
Is this required? It should be [implied](https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html#details) with the 2021 edition. Or did you see failures without it?